src/modules/ui/screen/screen.scss

```diff
@@ -6,71 +6,74 @@
   bottom: 0;
   left: 0;
 
+  display: flex;
+  flex-direction: column;
+
   width: 100%;
   height: 100%;
 
   background-color: black;
 
-  video {
-    position: relative;
-    z-index: 0;
+  justify-content: center;
+  align-items: center;
 
-    width: 100%;
-    height: 100%;
+  &.regularMode {
+    video {
+      width: 100%;
+      height: 100%;
+    }
   }
 
-  &.hiddenCursor {
-    cursor: none;
+  &.verticalVideo {
+    flex-direction: row;
+    &.fillMode {
+      video {
+        width: 100%;
+      }
+    }
+    &.blurMode {
+      video {
+        height: 100%;
+      }
+    }
   }
-}
-
-.screenTopBackground,
-.screenBottomBackground {
-  position: absolute;
-  z-index: 1;
-  right: 0;
-  left: 0;
-
-  display: none;
-  visibility: hidden;
-
-  height: 182px;
 
-  transition: opacity .2s, visibility .2s;
+  &.horizontalVideo {
+    flex-direction: column;
+    &.fillMode {
+      video {
+        height: 100%;
+      }
+    }
+    &.blurMode {
+      video {
+        width: 100%;
+      }
+    }
+  }
 
-  opacity: 0;
-  background-size: 100% 182px;
+  video {
+    position: relative;
+    z-index: 1;
 
-  &.visible {
-    display: block;
-    visibility: visible;
+    box-shadow: 0 0 20px rgba(0, 0, 0, .2);
+  }
 
-    opacity: 1;
+  &.hiddenCursor {
+    cursor: none;
   }
 }
 
-.screenTopBackground {
-  top: 0;
-
-  background-image: linear-gradient(
-  to top,
-  rgba(0, 0, 0, .0),
-  rgba(0, 0, 0, .03) 24%,
-  rgba(0, 0, 0, .15) 50%,
-  rgba(0, 0, 0, .3) 75%,
-  rgba(0, 0, 0, .4)
-  );
-}
+$canvas-padding-vertical: 20px;
```

Comment:
mb move `$canvas-padding` to inline styles and use `CANVAS_BACKGROUND_PADDING` constant?